joschahenningsen/TUM-Live
# SeekBar Marker

### Motivation and Context
Add more information to the seekbar

### Description
Add Sections to the seekbar to give a quick overview about the different video segments.

### Steps for Testing
Prerequisites:
- 1 Student
- 1 Livestream

1. Log in
2. Navigate to a Livestream with sections and heatmap
3. Check if sections are represented over the seekbar

### Screenshots
![Bildschirmfoto 2023-03-11 um 18 19 02](https://user-images.githubusercontent.com/17506411/224503383-1da282b1-36a7-4129-b97d-b270d0f7edf3.png)
